core-backend/wasmi/src/funcs/mod.rs

```diff
@@ -56,7 +56,7 @@ pub enum FuncError<E: Display> {
     #[display(fmt = "{_0}")]
     Core(E),
     #[display(fmt = "Runtime Error")]
-    HostError,
+    WrongInstrumentation,
```

Comment:
fmt message and error itself are incompatible now

Reply:
changed